src/zimdump.cpp

```diff
@@ -49,7 +48,22 @@
 
 #define ERRORSDIR "_exceptions/"
 
-std::wstring_convert<std::codecvt_utf8_utf16<wchar_t>> converter;
+
+#ifdef _WIN32
+std::wstring utf8ToUtf16(const std::string& string) {
+  auto size = MultiByteToWideChar(CP_UTF8, 0,
+                path.c_str(), -1, nullptr, 0);
+  auto wdata = std::wstring(size, 0);
+  auto ret = MultiByteToWideChar(CP_UTF8, 0,
+                path.c_str(), -1, wdata.get(), size);
```

Comment:
In [my previous review comment](https://github.com/openzim/zim-tools/pull/376#discussion_r1367015751), the main concern was that - as a result of copy-paste-modify - the variable name  `path` became out of sync with the function parameter name (which was changed to `string`). After you fix it and the CI win32 build passes there is no need to request another review.